tests/lib/utils

```diff
@@ -52,3 +52,12 @@ function improver_compare_output {
   [[ "$status" -eq 0 ]]
   [[ "$output" =~ "are identical." ]]
 }
+
+function improver_check_recreate_kgo {
+  # If RECREATE_BATS_KGO set, copy output files created by test to a
+  # new set of KGO located at the path specified by RECREATE_BATS_KGO
+  if [ -n "$RECREATE_BATS_KGO" ]; then
+    mkdir -p "$RECREATE_BATS_KGO/${2%/*}"
```

Comment:
I still reckon basename is less confusing

Reply:
This needs to be dirname as we want the directory path, not the file. Change made.